awx/api/serializers.py

```diff
@@ -1458,6 +1458,11 @@ class Meta:
             'organization',
             'scm_update_on_launch',
             'scm_update_cache_timeout',
+            'playbook_integrity_enabled',
+            'playbook_integrity_public_key',
+            'playbook_integrity_signature_type',
```

Comment:
Should we add some sort of validation for this field? What values are allowed? From reading your docs it looks like `sigstore` and `gpg` are supported.

Reply:
@shanemcd yes, validation is needed. We have some validation only in [collection side](https://github.com/IBM/playbook-integrity-collection/blob/main/plugins/module_utils/sign.py#L33-L43), but should have validation at this level. We will fix this. @hirokuni-kitahara

Reply:
@shanemcd @yuji-watanabe-jp Thank you, I will add validation for the field.